From 3bb910503bb7cbc20105f0a302db400f04436d2a Mon Sep 17 00:00:00 2001
From: ikerexxe <ipedrosa@redhat.com>
Date: Tue, 18 Aug 2020 11:45:18 +0200
Subject: [PATCH] man: clarify AD certificate rule

Clarify AD specific certificate rule example by changing userPrincipal to
userPrincipalName. Moreover, match the subject principal name in the
example with the rule name.

Resolves:
https://github.com/SSSD/sssd/issues/5278

Reviewed-by: Sumit Bose <sbose@redhat.com>
---
 src/man/sss-certmap.5.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/man/sss-certmap.5.xml b/src/man/sss-certmap.5.xml
index 10343625e..09aec997c 100644
--- a/src/man/sss-certmap.5.xml
+++ b/src/man/sss-certmap.5.xml
@@ -487,7 +487,7 @@
                         sign.
                     </para>
                     <para>
-                        Example: (|(userPrincipal={subject_principal})(samAccountName={subject_principal.short_name}))
+                        Example: (|(userPrincipalName={subject_nt_principal})(samAccountName={subject_nt_principal.short_name}))
                     </para>
                     </listitem>
                 </varlistentry>
-- 
2.21.3

